app/AppKernel.php

```diff
@@ -66,7 +66,7 @@ public function registerContainerConfiguration(LoaderInterface $loader)
         if ($this->getEnvironment() === 'dev') {
             $loader->load(function (ContainerBuilder $container) {
                 // Check if webpack dev server is up before using it
-                @file_get_contents($container->getParameter('webpack_dev_server_base_url'));
+                @file_get_contents($_ENV['WEBPACK_DEV_SERVER_BASE_URL']);
```

Comment:
```suggestion
                @file_get_contents($container->getParameter('webpack_dev_server_base_url'));
```

I prefer to not directly access super globals in PHP (to avoid being able to change its value at runtime), I'd prefer to put the following in `config.yml`:

```yml
parameters:
    # ...
    webpack_dev_server_base_url: '%env(WEBPACK_DEV_SERVER_BASE_URL)%'
```